UPGRADE.md

```diff
@@ -0,0 +1,80 @@
+UPGRADE GUIDE FROM 1.x to 2.0
+=============================
+
+This is guide for upgrade from version 1.x to 2.0 for using the CLI tool.
+
+Config file
+-----------
+From now you can create new configuration file: `.php_cs.dist`. This file is prioritized over `.php_cs` configuration file. It is recommended to use first one and add second one to `.gitignore` to allow your contributors to have theirs own configuration file.
+
+Config and Finder classes
+-------------------------
+All off `Symfony\CS\Config\*` and `Symfony\CS\Finder\*` classes have been removed, instead use `Symfony\CS\Config` and `Symfony\CS\Finder`.
+
+For that reason one can not set config class by `--config` CLI argument, from now it is used to set configuration file. Thanks to this the `--config-file` CLI argument is no longer available.
+
+Rules and sets
+--------------
+To configure which fixers should be used one must now set rules and sets instead of fixers and level. This affects both configuration file and CLI arguments.
+
+The term of risky fixers was introduced. Risky fixer is a fixer that may change the meaning of code (like `StrictFixer` fixer, which will change `==` into `===`). No rules that are followed by risky fixers are run be default. One need to explicitly permit risky fixers to run them.
+
+Changes to rules
+----------------
+Rules that have been removed:
+
+Name | Reason
+---- | ------
+phpdoc_var_to_type | use phpdoc_type_to_var instead, which follows PSR-5
```

Comment:
should we really remove it entirely ? Projects may not want to change their coding standards yet (especially as PSR-5 is still a draft), so the var_to_type way should still be possible IMO (even if you disable it by default)